`DashProxy.hijack` is not working anymore

As of the PR https://github.com/plotly/dash/pull/1876, `DashProxy.hijack(...)` raises an `AttributeError`

```
>       _app.hijack(app)

../../conftest.py:71: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/foo/lib/python3.8/site-packages/dash_extensions/enrich.py:134: in hijack
    app.config.update(self.config)
/foo/lib/python3.8/site-packages/dash/_utils.py:116: in update
    self[k] = v
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = {'name': 'conftest', 'assets_folder': '/src/assets', 'assets_url_path': 'assets', '...llbacks': False, 'show_undo_redo': False, 'extra_hot_reload_paths': [], 'title': 'Dash', 'update_title': 'Updating...'}
key = 'name', val = 'app'

    def __setitem__(self, key, val):
        if key in self.__dict__.get("_read_only", {}):
>           raise AttributeError(self._read_only[key], key)
E           AttributeError: ('Read-only: can only be set in the Dash constructor', 'name')

/foo/lib/python3.8/site-packages/dash/_utils.py:105: AttributeError
```

It seems that this is due to trying to `.update()` the whole config of a new app with the old one while `dash` restricts setting "read-only" keys in the dictionary.

Should be pretty easy to fix, IMO. I will prepare a PR if I have time.

P.S. See [this comment](https://github.com/thedirtyfew/dash-extensions/issues/79#issuecomment-1066785380) to see my setup for `.hijack()`.

I'm using `dash==2.3.1`

--- a/dash_extensions/enrich.py
+++ b/dash_extensions/enrich.py
@@ -147,7 +147,8 @@ class DashProxy(DashBase):
 
     def hijack(self, app: DashBase):
         # Change properties.
-        app.config.update(self.config)
+        readonly_props = app.config.__dict__.get("_read_only", {})
+        app.config.update({k: v for k, v in self.config.items() if k not in readonly_props})
         app.title = self.title
         app.index_string = self.index_string
         # Inject layout.

